_includes/master/reqs-sys.md

```diff
@@ -31,19 +31,24 @@ to connect to, we provide instructions in the [installation documentation](./ins
 
 ## Network requirements
 
-{{site.prodname}} requires the network to allow the following types of traffic.
-
-| Traffic | Protocol | Port |
-| ------- | -------- | ---- |
-| BGP     | TCP      | 179  |
-| IPIP\*  | 4        | n/a  |
-
-\* {% if include.orch == "OpenStack" %} If your compute hosts connect directly
-   and don't use IPIP, you don't need to allow IPIP traffic. {% else %} Our
-   manifests enable IPIP by default.  If you disable IPIP, you won't need to
-   allow IPIP traffic. {% endif %} Refer to [Configuring
-   IP-in-IP](../../usage/configuration/ip-in-ip) for more information.
-
-> **Tip**: On GCE, you can allow this traffic using firewall rules. In AWS, use
-> EC2 security group rules.
-{: .alert .alert-success}
+Ensure that your hosts and firewalls allow the following traffic.
+
+| Configuration                                                                         | Host                | Connection type | Port/protocol                                                 |
+|---------------------------------------------------------------------------------------|---------------------|-----------------|---------------------------------------------------------------|
+| {{site.prodname}} networking                                                          | All                 | Bidirectional   | TCP 179                                                       |
+| {{site.prodname}} networking in IP-in-IP mode (default mode)                          | All                 | Bidirectional   | IP-in-IP, often represented by its protocol number `4`        |
+{%- if include.orch == "Kubernetes" %}
+| flannel networking                                                                    | All                 | Bidirectional   | UDP 4789 (default)                                            |
+| etcd datastore                                                                        | etcd hosts          | Incoming        | Varies per etcd configuration                                 |
+| {{site.prodname}} networking, Kubernetes API datastore, cluster of more than 50 nodes | Typha agent hosts   | Incoming        | TCP 5473 (default)                                            |
```

Comment:
I think it's awkward and not really correct to say >50 nodes here - this applies if you're running Typha, independent of cluster size.

Reply:
The requirements are just intended to allow people to accomplish the installation without difficulty or error. This phrasing matches the phrasing used later on in the install procedure and also with text that we use elsewhere. It is better if it all matches.

Reply:
I understood that was the intention, but I still think it is confusing for users and strictly incorrect. Elsewhere, we try to be technically accurate, and I think we should do so here as well.